### PR TITLE
Spanish cards

### PR DESCRIPTION
**Pull request by Cam-McDonnell**
Displays Spanish cards based on user input

### CODE REVIEW

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -62,6 +67,12 @@
     });
 
     var game;
+    var isSpanish = 0;
+    function isSpain(input){
+        isSpanish = input;
+    }
+
```
Comment (jonathanharijanto) [inline src/main/java/views/AcesUp/AcesUp.flt.html:74]: Is it okay if I make some adjustment later? Overall, the logic is similar to what we discussed in group meeting.

**Comment by jonathanharijanto** [comment]
I wrote an input in the code directly. It is basically about me trying to adjust several code later.

File: src/main/java/views/AcesUp/AcesUp.flt.html (hunk 1 above)
Comment (Cam-McDonnell) [inline src/main/java/views/AcesUp/AcesUp.flt.html:74]: Ya this was more of a temporary way to test functionality

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 2]
```diff
@@ -29,6 +29,11 @@
```
Comment (jonathanharijanto) [inline src/main/java/views/AcesUp/AcesUp.flt.html:29]: I changed the names of the card because it still called clubs hearts spades and diamonds.

**Pull request merged**